### ISSUE TITLE
Is http2 allowed to be passed to AWSXRay.captureHTTPsGlobal?

### ISSUE THREAD

**Issue by kitsunde** [aws/aws-xray-sdk-node#579]
The type definition for `captureHTTPsGlobal` does not include http2. Is it supported?

```ts
export function captureHTTPsGlobal(mod: typeof https | typeof http, downstreamXRayEnabled?: boolean, subsegmentCallback?: httpSubsegmentCallback): void;
```

**Comment by carolabadeer** [comment]
Hi @kitsunde, 
The X-Ray Node SDK does not currently support instrumentation for `HTTP2`, which is why it is not included in the type definitions for `captureHTTPs` functions. However, there is an [open feature request](https://github.com/open-telemetry/opentelemetry-js/issues/2667) for support for HTTP2 in the opentelemetry-js repo, which I have followed up on. 
I would recommend keeping an eye on that issue to see if/when opentelemetry plans to support `HTTP2` instrumentation, and we are always happy to review any feature contributions if you are open to making a PR!

**Comment by kitsunde** [comment]
Right. Is the idea that X-Ray will not continue to be built upon beyond maintenance? The language and in the repo is a bit vague.

**Comment by carolabadeer** [comment]
HTTP2 instrumentation is not currently on our roadmap for instrumentation support, but we are still welcoming and reviewing any feature contributions for the X-Ray SDKs :)
